projects/shidoswap/index.js

```diff
@@ -0,0 +1,18 @@
+const { get } = require('../helper/http')
+const TVL_ENDPOINT = 'https://swagger.shidoscan.com/cosmos/staking/v1beta1/pool';
```

Comment:
we cant count chain staking as tvl, is shidodex uni v2 fork? if so, can you share factory address and chain rpc details?